docs/payload.md

```diff
@@ -0,0 +1,111 @@
+<!-- Generator: Widdershins v4.0.1 -->
+
+<h1 id="timecard">TimeCard v0.1.0</h1>
+
+The TimeCard container exposes a number of models to clients. The TimeEntry is a key model from which most others hang
+
+# Schemas
+
+<h2 id="tocS_Note">Note</h2>
+<!-- backwards compatibility -->
+<a id="schemanote"></a>
+<a id="schema_Note"></a>
+<a id="tocSnote"></a>
+<a id="tocsnote"></a>
+
+```json
+{
+  "content": "string",
+  "date": "current date and time",
+  "createdAt": "2019-08-24T14:15:22Z",
+  "authorId": 0
+}
+
+```
+
+A note is used to carry arbitrary textual information about a date. Notes are immutable.
+
+### Properties
+
+|Name|Type|Required|Restrictions|Description|
+|---|---|---|---|---|
+|content|string|true|none|Holds the note's text|
+|date|string(date)|true|none|The date that the note is associated with|
+|createdAt|string(date-time)|false|none|Assigned by the TimeCard container when the note is persisted|
+|authorId|number|false|none|The id of the Person who wrote the note's content|
+
+<h2 id="tocS_TimePeriodType">TimePeriodType</h2>
+<!-- backwards compatibility -->
+<a id="schematimeperiodtype"></a>
+<a id="schema_TimePeriodType"></a>
+<a id="tocStimeperiodtype"></a>
+<a id="tocstimeperiodtype"></a>
+
+```json
+{
+  "timePeriodTypeId": 0,
+  "name": "string",
+  "valueType": "date"
+}
+
+```
+
+A way to categorise time periods (e.g. a shift, a standard rest day). Depending on the type of time period the actual time value that should be recorded against that period varies. For example to record a shift time period two data points are needed - a shift start datetime and and a shift end datetime. Contrast this with a standard rest day where only a single date is needed.
+
+### Properties
+
+|Name|Type|Required|Restrictions|Description|
+|---|---|---|---|---|
+|timePeriodTypeId|number|false|none|the identifier for this TimePeriodType|
+|name|string|true|none|the display name of this TimePeriodType|
+|valueType|string|true|none|The value that a time period of this type should contain|
+
+#### Enumerated Values
+
+|Property|Value|
+|---|---|
+|valueType|date|
+|valueType|datetime|
+
+<h2 id="tocS_TimeEntry">TimeEntry</h2>
+<!-- backwards compatibility -->
+<a id="schematimeentry"></a>
+<a id="schema_TimeEntry"></a>
+<a id="tocStimeentry"></a>
+<a id="tocstimeentry"></a>
+
+```json
+{
+  "timeEntryId": 0,
+  "version": 0,
+  "ownerId": 0,
+  "actualStartTime": "2019-08-24T14:15:22Z",
+  "actualEndTime": "2019-08-24T14:15:22Z",
+  "plannedStartTime": "2019-08-24T14:15:22Z",
+  "plannedEndTime": "2019-08-24T14:15:22Z",
+  "shiftType": "string",
+  "timePeriodType": {
+    "timePeriodTypeId": 0,
+    "name": "string",
```

Comment:
Should this be an actual value instead of data type `string`? e.g. `"name": "Scheduled Rest Day"`.
There are a few occurrences in this file (e.g. shiftType) where the data type `string` is used instead of an actual sample value, which would be more helpful

Reply:
@sureshmanga1, @stephenkaye-ho and I discussed this point during the implementation phase. Unfortunately we can't make this more strongly typed (which would be nice) because the time period type will ultimately become a config driven. Different tenants and possibly different groups within the same tenant want the ability to have their own time period types. 

We don't want to have to rebuild the source code and redeploy every time a change is needed to time period type hence leaving it as a string

Reply:
Sorry, I didn't mean making it more strongly typed; string type seems good enough. I meant the example provided should be meaningful, for example: `"name: "Scheduled Rest Day"` instead of `"name": "string"`

Reply:
Discussed with @taomoh - because payload.md (and the examples it contains) are generated from a model the content can't really be edited.